Taskfile.yml

```diff
@@ -258,6 +262,17 @@ tasks:
         -target-backend=mongodb
         -no-xfail={{.NO_XFAIL}}
 
+  convert-test-results:
+    desc: "Convert test results to JUnit XML format"
+    dir: integration
+    cmds:
+      - >
+        ../bin/go-junit-report -parser=gojson -iocopy
+        -in {{.IN_FILE}}
+        -out {{.OUT_FILE}}
+    requires:
+      vars: [IN_FILE, OUT_FILE]
```

Comment:
Could we have default values? Let's say I want to run it locally, it's not trivial to know what values to set.

Reply:
What would be a default value for those? This helper function is used for both `postgresql` and `mongodb` files. I don't think it has any deafault usage (similary to the `docker-build` task for example).